### PR TITLE
TV Show Webpage receiving and sending data to and from API

### PR DESCRIPTION
**Pull request by Haywayaheadshot**
### What?

My partner and I created a front-end environment using webpack and create the homepage to display tv shows.

### Why?

To enable users like their favorite shows and comment under their distaste if they have any.

### How?
- Installed webpack.
- Installed Linters.
- Followed proper gitflow.
- We wrote the HTML for the header.
- We imported a image to serve as the logo of the page.
- We dynamically added the logo to the page using JavaScript.
- We added a footer to the page.
- Add main section to HTML body.
- Display shows by fetching data from tvmaze REST API.
- We created a pop up feature for the comments.
- We created a comments display.
- We created a like count.
- We created a like count display.
- We tested for the likes and comments count.
- We styled up the page.

### CODE REVIEW

File: src/modules/TotalMovieCounter.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+const n = 21;
+
+export default n;
```
Comment (ShadyShawkat) [inline src/modules/TotalMovieCounter.js]: - [x] Kindly implement a method that calculates the number of items fetched from the API, instead of returning a placeholder number, as per [this](https://github.com/microverseinc/curriculum-javascript/blob/main/group-capstone/js_capstone.md#:~:text=you%20will%20create%20a%20specific%20function%20to%20calculate%20these%20numbers%20in%20each%20page.) requirement.

File: src/__test__/counters.test.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+      {},
+      {},
+      {},
+      {},
+      {},
+      {},
+    ];
+    expect(shows.length).toBe(20);
+  });
+});
```
Comment (ShadyShawkat) [inline src/__test__/counters.test.js]: - [x] Kindly add more tests for the methods that calculate the counters, as per [this](https://github.com/microverseinc/curriculum-javascript/blob/main/group-capstone/js_capstone.md#:~:text=Even%20if%20the%20API%20gives%20you%20these%20numbers%2C%20you%20will%20create%20a%20specific%20function%20to%20calculate%20these%20numbers%20in%20each%20page.%20These%20count%20functions%20need%20to%20be%20covered%20with%20unit%20tests%20using%20Jest.) requirement.

- Please remember to test all test cases and edge cases like if the method returned 0.

- [HINT] Kindly refrain from implementing redundant tests like the last test which doesn't test any functionality in the app. Also, please test the methods that you are to implement which calculate the counters instead of testing the length of nodes.

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi Team, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
- All linters are passing ✔️ 
- Correct use of GitFlow ✔️ 

Image

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by descholar-ceo — APPROVED** [review]
Hi @Haywayaheadshot && @taldr27 🤝 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
👍🏽 Implemented all of the requested changes ✔️ 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Haywayaheadshot** [comment]
Thank you @descholar-ceo and @ShadyShawkat for reviewing our project.
